### PR TITLE
Proxy component, first major release, root frontend imports

### PR DESCRIPTION
**Pull request by markrawls**
(fireworks emoji here)

### CODE REVIEW

File: zygoat/components/frontend/dockerfile.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 from . import resources
 
 log = logging.getLogger()
-file_name = "Dockerfile"
+file_name = "Dockerfile.dev"
```
Comment (ambikads) [inline zygoat/components/frontend/dockerfile.py:13]: `.local` might be better?. If so, https://github.com/MetLifeLegalPlans/zygoat/pull/84 can be merged

Comment (markrawls) [reply zygoat/components/frontend/dockerfile.py]: Good call - merged and rebuilt this on top.

File: pyproject.toml [hunk 2]
```diff
@@ -16,6 +16,8 @@ virtualenv = "^20.0"
 redbaron = "^0.9.2"
 requests = "^2.23.0"
 rtd-poetry = "^0.1.0"
+importlib_metadata = "^1.7.0"
+importlib-metadata = "^1.7.0"
```
Comment (Ian-MacLeod) [inline pyproject.toml:20]: Do we need both of these?

Comment (markrawls) [reply pyproject.toml]: Nope, poetry is weird. Fixed!

**Pull request merged**